vars/sconsBuild.groovy

```diff
@@ -135,7 +135,19 @@ def call(Map config = [:]) {
         set_cwd = "cd ${config_target}\n"
     }
 
-    def scons_exe = 'scons'
+    String scons_exe = 'scons'
+    int sctest = sh label: 'probe for scons command',
+                    script: 'command -v scons',
+                    returnStatus: true
+    if (sctest != 0) {
+        sctest = sh label: 'probe for scons-3 command',
+                    script: 'command -v scons-3',
+                    returnStatus: true
+        if (sctest == 0) {
+            scons_exe = 'scons-3'
+        }
+    }
+
```

Comment:
All the dockerfiles have used 'scons' installed through pip for some months now, no change to support dockerfile building should be considering scons-3 at all.